lib/tapioca/commands/abstract_dsl.rb

```diff
@@ -36,6 +37,7 @@ def initialize(
         exclude:,
         file_header:,
         tapioca_path:,
+        skip_constant:,
```

Comment:
I think this could be optional with a default value?

```suggestion
        skip_constant: [],
```